.gitignore

```diff
@@ -1,4 +1,5 @@
 composer.phar
 composer.lock
 vendor/
-build/
\ No newline at end of file
+build/
+.idea/
```

Comment:
Please remove the `.idea` from this file. It should go in your global gitignore file.